codebuild/bin/install_openssl_3_0.sh

```diff
@@ -44,7 +44,7 @@ mkdir -p $INSTALL_DIR
 $CONFIGURE shared -g3 -fPIC              \
          no-md2 no-rc5 no-rfc3779 no-sctp no-ssl-trace no-zlib     \
          no-hw no-mdc2 no-seed no-idea enable-ec_nistp_64_gcc_128 no-camellia\
-         no-bf no-ripemd no-dsa no-ssl2 no-ssl3 no-capieng                  \
+         no-bf no-ripemd no-dsa no-ssl2 no-ssl3 no-capieng no-dtls          \
```

Comment:
can this be applied to other installations also?

Reply:
I think we can do it separately if needed. I'd prefer to focus on getting CI unblocked.

Reply:
the reason for this was the openssl3 unit tests fail inside of a docker container on dtls.  Could not reproduce it outside of docker.